src/actions/index.js

```diff
@@ -1,7 +1,6 @@
 /* eslint-disable promise/always-return */
 // eslint-disable-next-line no-redeclare
 
-import emailHelper from 'lib/emailHelper'
```

Comment:
can we delete this lib/emailHelper file?

Reply:
No we are still using it into the EmailModal to check the email format